backend/app/views/spree/admin/payment_methods/_form.html.erb

```diff
@@ -19,9 +19,9 @@
             <% if @object.preference_source.present? %>
               <%= t('spree.preference_source_using', name: @object.preference_source) %>
             <% elsif !@object.new_record? %>
-              <% @object.preferences.keys.each do |key| %>
-                <%= render "spree/admin/shared/preference_fields/#{@object.preference_type(key)}",
-                   form: f, attribute: "preferred_#{key}", label: t(key, scope: 'spree') %>
+              <% @object.admin_form_preference_field_types.each do |type| %>
```

Comment:
It seems to me this is a wrong change.  Given a class with preferable `:color, type: :string`, previously this would render the form with attribute `preferred_color`, whereas now it seems like it renders `preferred_string`, no?

Reply:
No, we're just not rendering a `type` here.

Reply:
Ah, yes. You are right. Dang. Thanks for catching this!